src/Altinn.Broker.Application/ConfigureResource/ConfigureResourceHandler.cs

```diff
@@ -130,4 +146,15 @@ private async Task<OneOf<Task, Error>> UpdateUseManifestFileShim(ResourceEntity
         await resourceRepository.UpdateUseManifestFileShim(resource.Id, useManifestFileShim, cancellationToken);
         return Task.CompletedTask;
     }
+
+    private async Task<OneOf<Task, Error>> UpdateExternalServiceCodeLegacy(ResourceEntity resource, string ExternalServiceCodeLegacy, CancellationToken cancellationToken)
+    {
+        await resourceRepository.UpdateExternalServiceCodeLegacy(resource.Id, ExternalServiceCodeLegacy, cancellationToken);
+        return Task.CompletedTask;
+    }
+    private async Task<OneOf<Task, Error>> UpdateExternalServiceEditionCodeLegacy(ResourceEntity resource, string ExternalServiceEditionCodeLegacy, CancellationToken cancellationToken)
+    {
+        await resourceRepository.UpdateExternalServiceEditionCodeLegacy(resource.Id, ExternalServiceEditionCodeLegacy, cancellationToken);
+        return Task.CompletedTask;
+    }
```

Comment:
Is this needed as its own function. Would be 1 line to use resourceRepository directly from the main function.